packages/api/db.ts

```diff
@@ -1,3 +1,5 @@
-import { PrismaClient } from './prisma/client'
+import { PrismaClient, Prisma } from './prisma/client';
 
-export const client = new PrismaClient()
\ No newline at end of file
+export { Prisma };
```

Comment:
If we export the Prisma namespace, with the client then we can get access to all of the generated Prisma types for all of the tables